lib/util.js

```diff
@@ -1,40 +1,58 @@
 "use strict";
 /* eslint-disable global-require */
+const isValidObject = require("./is-valid-object");
 
 const HAPI16 = 16;
 const HAPI17 = 17;
+const HAPI18 = 18;
 
 function hapiMajor() {
   let hapiPkg;
   try {
-    hapiPkg = require("hapi/package");
+    hapiPkg = require("@hapi/hapi/package");
   } catch (err) {
-    // ignore
+    hapiPkg = require("hapi/package");
   }
-  return hapiPkg ? +hapiPkg.version.split(".")[0] : HAPI16;
+
+  if (isValidObject(hapiPkg) && hapiPkg.version) {
+    return +hapiPkg.version.split(".")[0];
+  }
+
+  return HAPI16;
 }
-let _isHapi17 = hapiMajor() >= HAPI17;
+
+const hapiV = hapiMajor();
+let _isHapi17 = hapiV >= HAPI17;
```

Comment:
Should the getter for this also be named "isHapi17OrUp"? Or should this be a strict equality check?

Reply:
we can add `isHapi17OrUp` but can't change the existing `isHapi17`.
`isHapi17` should do `>=`, so existing modules will work with 18 w/o change, except an update on this module.

Reply:
Had the same doubt and i wanna leave it for discussion. I prefer strict equality check though.

Reply:
It seems clear to me based on Joel's comment that you did the right thing - folks are using the isHapi17 function as if it's isHapi17OrUp I suppose? As Joel said, you could optionally add a isHapi17OrUp function, but it would essentially have an identical implementation.